config/config-inyo/config_ignore.settings.yml

```diff
@@ -25,5 +25,6 @@ ignored_config_entities:
   - 'key.key.*'
   - 'xmlsitemap.xmlsitemap.*'
   - 'webform.webform.*'
+  - ~webform.webform.was_this_helpful
```

Comment:
Do we want to ignore this?

Reply:
No, it's the "ignore the ignore" trick we talked about some days ago

Reply:
Oh duh! Apparently I didn't look closely at it!